.eslintrc.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
     {
       files: ['**/*.ts'],
       extends: [
-        'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
this is no longer needed in the latest typescript-eslint

Reply:
then remove it? I'm not even sure why typescript-eslint exist in the first place.

Reply:
it was to override the eslint built-in rules - it was add to `@typescript-eslint/recommended` in later versions.

Reply:
Ok. After the printer is done I will finish kataw's own linter so we can swap out ESLint